src/text/chr.ts

```diff
@@ -0,0 +1,12 @@
+/**
+ * Returns the character correspondinf to the specified ASCII value
+ * @param {number} val The ASCII value to convert to string
+ * @returns The string with the character defined by the given ASCII value
+ */
+export const chr = (val: number): string => {
+  if (val >= 0 && val < 128) {
```

Comment:
Why should val be < 128 ?
ie: String.fromCharCode(164) -> "ñ"

Reply:
Fixed, same as above.